src/DynamoCoreWpf/Views/Preview/WatchTree.xaml

```diff
@@ -134,5 +291,49 @@
                 </HierarchicalDataTemplate>
             </TreeView.ItemTemplate>
         </TreeView>
+
+        <Grid Name ="ListLevelsDisplay" Grid.Row="1" >
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="15" />
+            </Grid.ColumnDefinitions>
+            
+            <!-- Shows counts of all items in List -->
+            
+            <TextBlock Name="ListItems" Grid.Column="2" Margin="0,3,0,7" Foreground="Gray" FontStyle="Italic"
+                       Visibility="{Binding IsCollection, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}">
+            {<Run Foreground="DarkRed" Text="{Binding NumberOfItemsWatchViewModel}" />}
+            </TextBlock>
+
+            <!-- Shows list@level labels in List -->
+            
+            <ListView Name="listLevelsView" 
+                      Grid.Column ="0" Margin="3,3,0,7" 
+                      Background="Transparent"
+                      Visibility="{Binding IsCollection, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}"
+                      BorderThickness="0">
+
+                <ListView.ItemsPanel>
+                    <ItemsPanelTemplate>
+                        <StackPanel Orientation="Horizontal"></StackPanel>
+                    </ItemsPanelTemplate>
+                </ListView.ItemsPanel>
+
+                <ListView.ItemTemplate>
+                    <DataTemplate>
+                        <WrapPanel Orientation="Horizontal">
+                            <TextBlock FontSize="9" FontFamily="Consolas">
+                                <Run Foreground="Gray">@</Run>
+                            </TextBlock>
+                            <TextBlock Name="indivListLevel" FontFamily="Consolas" FontSize="9">
+                                L<Run Foreground="Black" Text="{Binding level}"></Run>
```

Comment:
Change Text="{Binding}" probably could remove `Level` class. See: http://stackoverflow.com/questions/9391746/how-can-i-data-bind-a-list-of-strings-to-a-listbox-in-wpf-wp7